python/paddle/utils/lazy_import.py

```diff
@@ -32,9 +32,10 @@ def try_import(module_name):
         mod = importlib.import_module(module_name)
         return mod
     except ImportError:
-        err_msg = (
-            "Failed importing {}. This likely means that some paddle modules "
-            "require additional dependencies that have to be "
-            "manually installed (usually with `pip install {}`). ").format(
-                module_name, install_name)
+        if err_msg is None:
+            err_msg = (
+                "Failed importing {}. This likely means that some paddle modules "
+                "require additional dependencies that have to be "
+                "manually installed (usually with `pip install {}`). ").format(
+                    module_name, install_name)
```

Comment:
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/6836917/197105835-bfab9bc2-bb58-4e07-a382-1d5c353ce1f4.png">

<img width="717" alt="image" src="https://user-images.githubusercontent.com/6836917/197105770-d9d3629c-4e18-4fd6-9ea7-9570299f5686.png">
The change here doesn't pass the python coverage test, could you add a unit-test or revert the change (use the default error message)?